### PR TITLE
CAL-1471:Fix Progress bar display issue

### PR DESCRIPTION
**Pull request by Jihed525**

### CODE REVIEW

File: platform-ui-skin/src/main/webapp/skin/less/customStyle.less [hunk 1]
```diff
@@ -1415,9 +1415,8 @@ p, .postContent, .uiWikiContentDisplay{
 /* Enterprise skin-Progress bar*/
 .progress,
 .progress.progessStart {
-  height: 10px;
-  margin-top: 5px;
-  box-shadow: none;
+  height: 18px;
```
Comment (ahamdi) [inline platform-ui-skin/src/main/webapp/skin/less/customStyle.less]: Height should be 20px

File: platform-ui-skin/src/main/webapp/skin/less/customStyle.less [hunk 2]
```diff
@@ -1415,9 +1415,8 @@ p, .postContent, .uiWikiContentDisplay{
...
+  border-radius: 25px;
```
Comment (ahamdi) [inline platform-ui-skin/src/main/webapp/skin/less/customStyle.less]: Why did you add this property ?

Comment (Jihed525) [reply platform-ui-skin/src/main/webapp/skin/less/customStyle.less]: > Why did you add this property ?

UI enhancement, no?

File: platform-ui-skin/src/main/webapp/skin/less/customStyle.less [hunk 3]
```diff
@@ -1415,9 +1415,7 @@ p, .postContent, .uiWikiContentDisplay{
 /* Enterprise skin-Progress bar*/
 .progress,
 .progress.progessStart {
-  height: 10px;
-  margin-top: 5px;
-  box-shadow: none;
```
Comment (ahamdi) [inline platform-ui-skin/src/main/webapp/skin/less/customStyle.less]: Please change just the needed CSS rule as others may be introduced for other contexts.
